src/Cameras/babylon.deviceOrientationCamera.ts

```diff
@@ -1,22 +1,10 @@
 module BABYLON {
     // We're mainly based on the logic defined into the FreeCamera code
     export class DeviceOrientationCamera extends FreeCamera {
-        //-- Begin properties for backward compatibility for inputs

```

Comment:
Ah! Sory, I was commenting on something different.
The function exist in FreeCamera, and only return values in case mouse / keyboard is attached. the setters are only set when there is a mouse/keyboard. So I think this is actually better this way, because this way you could have keyboard movement and device orientation in a nicer way.